docs/api/web-frame-main.md

```diff
@@ -71,8 +71,30 @@ or `undefined` if there is no WebFrameMain associated with the given IDs.
 Process: [Main](../glossary.md#main-process)<br />
 _This class is not exported from the `'electron'` module. It is only available as a return value of other methods in the Electron API._
 
+### Instance Events
+
+#### Event: 'destroyed'
+
+Returns:
+
+* `event` Event
+
+Emitted when the frame has been destroyed.
```

Comment:
Maybe add a note here about what can cause a frame to be destroyed?

Reply:
f56aa0c